src/scripts/discord.js

```diff
@@ -25,22 +25,22 @@ const observer = (event, arg) => {
       rpc.setActivity({
         ...idleStatus,
         ...{
-          details: `Listening to ${mediaInfoModule.mediaInfo.title}`,
```

Comment:
I think we should keep the "Listening to" text here, it looks real good in Discord 😄